bin/connector_config/Generate.hs

```diff
@@ -217,6 +226,11 @@ emitConnectorConfigH emitLn = do
     emitLn $ "#define PrologueId (" ++ show (fromEnum $ Prologue) ++ ")"
     emitLn $ "#define EndgameId (" ++ show (fromEnum $ Endgame) ++ ")"
     emitLn $ "#define VictoryId (" ++ show (fromEnum $ Victory) ++ ")"
+    forM_ [24 .. 31] $ \i ->
+        emitLn $ "#define Tower" ++ show (i - 23) ++ "Id (" ++ show (fromEnum $ T i) ++ ")"
+    forM_ [32 .. 40] $ \i ->
+        emitLn $ "#define Ruins" ++ show (i - 31) ++ "Id (" ++ show (fromEnum $ R i) ++ ")"
```

Comment:
We should be doing the `i-31` in `toEnum` and `fromEnum` instead of here.

Reply:
The lines are adjusted a bit to fix range errors, but the ``i-31`` is still around here as even though it's done in ``to`` and ``fromEnum`` leaving it as ``i`` just results in Tower 24 anyway.